src/mbed_os_tools/test/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -28,7 +28,11 @@ def __init__(self, name, config, importer=__import__):
         self.platform_name = config.get('platform_name', None)
         self.baudrate = config.get('baudrate', DEFAULT_BAUD_RATE)
         self.image_path = config.get('image_path', None)
-        self.allocate_requirements = {"platform_name": self.platform_name}
+        self.allocate_requirements = {
```

Comment:
could you add one unit test for this change to avoid regression in future?

Reply:
actually existing con_primitive_remote tests detect this change because of red results - you have to fix tests to get green results :)

Reply:
tests pass locally now, but not here, it seems.